src/test/scripts/helm_install.sh

```diff
@@ -47,6 +47,12 @@ helm install -n "${TARGET_NAMESPACE}" --wait \
    --version "$POSTGRES_CHART_VERSION" \
    bitnami/postgresql > $LOG_DOWNLOAD_DIR/helm_install_log.txt
 
+if [ -n "$2" ]
```

Comment:
All of the other parameters to this script come in via the `helm_parameters` file, this one shouldn't be any different. There's too many of them to add as command line aguments.